.github/workflows/osc-logs-CI.yml

```diff
@@ -0,0 +1,31 @@
+name: osc-logs-CI
+
+on:
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.18
+
+    - name: osc-logs builds
+      run: go build -v ./main.go
+
+    - name: install jq
+      run: apt-get -y install jq
```

Comment:
I think you would also need to update so `apt-get update -y && apt-get -y install jq`